src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -174,6 +176,14 @@ function FloatingActionButtonAndPopover({onHideCreateMenu, onShowCreateMenu}: Fl
         selector: hasSeenTourSelector,
     });
 
+    const shouldRedirectToOD = useMemo(() => {
+        const groupPolicies = Object.values(allPolicies ?? {}).filter((policy) => ReportUtils.isGroupPolicy(policy?.type ?? ''));
+        if (groupPolicies.length === 0) {
+            return false;
+        }
+        return groupPolicies.every((policy) => !policy?.isPolicyExpenseChatEnabled);
```

Comment:
Base on description, `none of the group policies they are a member of have isPolicyExpenseChatEnabled=true
` I think we should update condition due to we have a case isPolicyExpenseChatEnabled is a string

![Screenshot 2024-11-08 at 12 01 43](https://github.com/user-attachments/assets/1549271d-50d6-449a-aee6-79ccbac14967)


```suggestion
        return !groupPolicies.some((policy) => policy?.isPolicyExpenseChatEnabled === true);
```